packages/angular/README.md

```diff
@@ -72,3 +72,35 @@ Ionic developers can access this by importing from `@ionic/angular/standalone`.
 This is where the lazy loaded component implementations live.
 
 Ionic developers can access this by importing from `@ionic/angular`.
+
+## Building
+
+1. Install dependencies in `@ionic/core`:
+
+```shell
+cd core && pnpm install
+```
+
+2. Build `@ionic/core`. This will generate Angular component bindings in the `packages/angular` directory:
+
+```shell
+pnpm build
+```
+
+> [!WARNING]
+> If you are receiving errors about missing packages, you may need to run `pnpm install --force` to force the installation of the missing packages.
+
+3. Install dependencies in `@ionic/angular`:
+
+```shell
+cd packages/angular && pnpm install
```

Comment:
`pnpm install` will only be needed the first time to establish the symlink between core and angular. I wonder if we just recommend `pnpm install -r` as the first step to install all dependencies in the workspace and that will simplify the flow for development to just building the specific packages needed.

Reply:
I thought about that too. My hesitation is if a developer is only wanting to work in Angular, should they also have to wait for the dependencies like `@ionic/react`? It doesn't take very long do the installs though, so I'm open to using `pnpm install -r`.

Reply:
For record: We are leaving it as is: https://github.com/ionic-team/ionic-framework/pull/28958#pullrequestreview-1860465599.